api/test/wfl/unit/spec_test.clj

```diff
@@ -47,11 +47,8 @@
       (is (test? ::all/labels [label]) (format "failed: %s" label)))))
 
 (deftest test-watchers-spec
-  (let [valid?   s/valid?
-        invalid? (comp not s/valid?)] ;; i die
-    (doseq [[test? email]
-            [[valid?   "hornet-eng@broadinstitute.org"]
-             ;; From tbl: https://www.netmeister.org/blog/email.html
-             [valid?   "'*+-/=?^_`{|}~#$@[IPv6:2001:470:30:84:e276:63ff:fe72:3900]"]
-             [invalid? "foo"]]]
-      (is (test? ::all/watchers [email]) (format "failed: %s" email)))))
+  (let [valid?   (partial s/valid? ::all/watchers)
+        invalid? (complement valid?)]
+    (is (valid?   ["hornet-eng@broadinstitute.org"]))
+    (is (valid?   ["'*+-/=?^_`{|}~#$@[IPv6:2001:470:30:84:e276:63ff:fe72:3900]"]))
+    (is (invalid? ["foo"]))))
```

Comment:
This previously printed the email that failed in the log - that was helpful for immediately recognising what was wrong in PRs.